tests/installation/partitioning_lvm.pm

```diff
@@ -100,15 +100,15 @@ sub run {
             assert_screen "inst-select-root-disk";
             send_key 'alt-n';
         }
-        if (leap_version_at_least('15.0')) {
+        if (leap_version_at_least('15.0') || !is_sle) {
```

Comment:
Hm, wouldn't that also match on Leap 42.3? How about checking `!(is_sle && !sle_version_at_least '15') && !(is_leap && !leap_version_at_least 15.0')`?

Reply:
you're assuming 42.3 will get storage-ng? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning_lvm.pm#L90

well, it's impossible to have storage-ng in 42.3...

Reply:
No, of course Leap 42.3 will not get storage-ng. I understood the code better now. There is already a check for "is_storage_ng" so the only thing that we need to do is to define different shortcuts for opensuse and SLE based on widgets that are shown on the screen or not. I assume the problem could be a "release notes" button which might take the hotkey.

So `if (is_opensuse)` should be the way to go.